.github/runtime_specs/westend.json

```diff
@@ -0,0 +1,17 @@
+{
+    "pallets": {
+      "1": {
```

Comment:
Could we go by pallet name?

Reply:
In this first version, I kept things simple and I am not interpreting the specs. This is why they must match 100% the format of the actuall metadata.

That could be improvement in the future but I would rather get started simple to have this check in place asap.

The benefit of the current option as well is that we can pretty much use the current metadata as spec, that would be may too strict but it works as a starting point.